core/src/main/java/org/geysermc/geyser/translator/protocol/java/JavaUpdateRecipesTranslator.java

```diff
@@ -81,11 +82,24 @@ public class JavaUpdateRecipesTranslator extends PacketTranslator<ClientboundUpd
             MultiRecipeData.of(UUID.fromString("602234e4-cac1-4353-8bb7-b1ebff70024b"), ++LAST_RECIPE_NET_ID) // Map locking
     );
 
+    private static final List<String> NETHERITE_TRANSFORM = List.of(
+            "minecraft:netherite_sword",
+            "minecraft:netherite_shovel",
```

Comment:
`NETHERITE_ITEMS`? These don't have the `_smithing` suffixed yet.

Reply:
Or `NETHERITE_UPGRADES`? I don't know.

Reply:
maybe "netherite_recipes"?

Reply:
not a recipe though, its the result